src/symbol/projection.js

```diff
@@ -276,11 +290,19 @@ function placeGlyphsAlongLine(symbol,
     return {};
 }
 
+function projectTruncatedLineSegment(previousTilePoint: Point, currentTilePoint: Point, previousProjectedPoint: Point, minimumLength: number, projectionMatrix: mat4) {
+    const projectedUnitVertex = project(previousTilePoint.add(previousTilePoint.sub(currentTilePoint)._unit()), projectionMatrix).point;
```

Comment:
Would it be possible for `previousTilePoint + unitVector(previousTilePoint - currentTilePoint)` to fall behind the camera, if `previousTilePoint` happened to be within one unit of the camera plane?  If so, would that cause a similar problem to the one we're solving here?